packages/wallet/frontend/src/components/userCards/CardButtonMenu.tsx

```diff
@@ -0,0 +1,59 @@
+import { Button } from '@/ui/Button'
+import { Eye, EyeCross, Snow, Trash } from '../icons/CardButtons'
+import { useState } from 'react'
+import { useCardContext } from '@/lib/context/card'
+
+export const CardButtonMenu = () => {
+  const [isDetailed, setIsDetailed] = useState(false)
+  const [isFrozen, setIsFrozen] = useState(false)
```

Comment:
this state will probably be removed, as the frozen flag will come from GateHub endpoint.